Code/Framework/AzCore/AzCore/std/parallel/binary_semaphore.h

```diff
@@ -70,11 +70,11 @@ namespace AZStd
         bool try_acquire_until(const chrono::time_point<Clock, Duration>& abs_time)
         {
             auto timeNow = chrono::system_clock::now();
-            if (timeNow >= absTime)
+            if (timeNow >= abs_time)
             {
                 return false; // we timed out already!
             }
-            auto deltaTime = absTime - timeNow;
```

Comment:
What on earth was this doing before? What `absTime` did this resolve to? Or was this function never called, so the function template was never instantiated, so it didn't cause a compile error?

Reply:
wasn't being used so MSVC and Clang don't compile it
Seems that `/permissive-` does compile it